packages/cli/src/commands/locations/rooms/delete.ts

```diff
@@ -33,12 +31,12 @@ export default class RoomsDeleteCommand extends SelectingAPICommand<Room> {
 		const { args, argv, flags } = this.parse(RoomsDeleteCommand)
 		await super.setup(args, argv, flags)
 
-		const roomsPromise = this.getRoomsByLocation(flags['location-id'])
```

Comment:
I actually like the old name better here since this variable is only a promise of rooms, not a list of actual rooms. That said, I also don't see any reason we couldn't move the `await` to this line since we call `await` every other place we reference it.